baseapp_payments/serializers.py

```
@@ -272,8 +272,10 @@ def create(self, validated_data):
             )
             return {"id": setup_intent["id"], "client_secret": setup_intent["client_secret"]}
         except Exception as e:
-            logger.error(f"Failed to create setup intent: {str(e)}")
-            serializers.ValidationError("An internal error has occurred. Please try again later.")
+            logger.exception(f"Failed to create setup intent: {str(e)}")
```

Ruff (0.15.13)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)